app/build.gradle.kts

```diff
@@ -21,7 +22,8 @@ android {
     }
 
     signingConfigs {
-        create("release") {
+        // Temporarily commented out release signing config for debug builds
```

Comment:
Any "Temporarily commented out" shouldn't pushed to a PRs. @aniruddha-adhikary  you should create a CONTRIBUTION guideline ASAP! ⚠️